### ISSUE TITLE
Error in LUT Output Window

### ISSUE THREAD

**Issue by gafter** [dotnet/roslyn#31272]
The following error occasionally pops up int the "Live Unit Testing" "Output" window:

``` none
[14:12:48.162 Info] Live Unit Testing started.
[14:13:16.078 Error] - FatalError - System.ArgumentException: Inconsistent language versions
Parameter name: syntaxTrees
   at Microsoft.CodeAnalysis.VisualBasic.VisualBasicCompilation.CommonLanguageVersion(ImmutableArray`1 syntaxTrees)
   at Microsoft.CodeAnalysis.VisualBasic.VisualBasicCompilation..ctor(String assemblyName, VisualBasicCompilationOptions options, ImmutableArray`1 references, ImmutableArray`1 syntaxTrees, ImmutableDictionary`2 syntaxTreeOrdinalMap, ImmutableDictionary`2 rootNamespaces, ImmutableArray`1 embeddedTrees, DeclarationTable declarationTable, VisualBasicCompilation previousSubmission, Type submissionReturnType, Type hostObjectType, Boolean isSubmission, ReferenceManager referenceManager, Boolean reuseReferenceManager, AsyncQueue`1 eventQueue)
   at Microsoft.CodeAnalysis.VisualBasic.VisualBasicCompilation.UpdateSyntaxTrees(ImmutableArray`1 syntaxTrees, ImmutableDictionary`2 syntaxTreeOrdinalMap, ImmutableDictionary`2 rootNamespaces, DeclarationTable declarationTable, Boolean referenceDirectivesChanged)
   at Microsoft.CodeAnalysis.VisualBasic.VisualBasicCompilation.AddSyntaxTrees(IEnumerable`1 trees)
   at Microsoft.CodeAnalysis.VisualBasic.VisualBasicCompilation.CommonAddSyntaxTrees(IEnumerable`1 trees)
   at Microsoft.CodeAnalysis.LiveUnitTesting.BuildManagement.CompilationComparer.GetUpdatedCompilation(Compilation compilation, CommandLineArguments arguments, Logger logger, BuildDifference& differences)
   at Microsoft.CodeAnalysis.LiveUnitTesting.BuildManagement.EmitHelper.<EmitAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.CodeAnalysis.LiveUnitTesting.BuildManagement.BuildService.<EmitSingleProjectAsync>d__51.MoveNext()
```

**Comment by sharwell** [comment]
@shyamnamboodiripad Should this be moved to your team?

**Comment by shyamnamboodiripad** [comment]
Yes my team should probably take the first stab on this. I've logged https://devdiv.visualstudio.com/DevDiv/_workitems/edit/732798 for this. I'm going ahead and closing this for now - will reactivate if this turns out to be a bug in the compiler.

@gafter could you share repro steps for when this happens?

**Comment by adraut** [comment]
@shyamnamboodiripad We are seeing this consistently if a test project references a project file in the legacy and that project in-turn references a project file in the new SDK format.  I added `<LangVersion>latest</LangVersion>` to all of the project files and that didn't fix it.  I deleted the `.vs\lut` folder, with VS closed and rebuilt the entire project before enabled LUT again.  Is there something else we should do?
